filament/CMakeLists.txt

```diff
@@ -325,6 +330,11 @@ target_link_libraries(${TARGET} PUBLIC filaflat)
 target_link_libraries(${TARGET} PUBLIC filabridge)
 target_link_libraries(${TARGET} PUBLIC image_headers)
 
+# iOS does not support OpenGL, and Android links directly against EGL
```

Comment:
Is it correct to say "iOS does not support OpenGL" , or is it a specific version of GL?

Reply:
I should rephrase this as "Filament on iOS does not yet support the OpenGL backend" (see my note to Romain below)